Gaijinizer patch/Map085.txt

```diff
@@ -205,9 +215,10 @@ being remodeled.#MTLed
 直前までどこかの孤児院にいたことや、
 アイドルにならなきゃって強く願ってたことは覚えてる。
 > CONTEXT: Map085/events/5/pages/1/56/Dialogue/enemy6
-But not all at all. I had been in an orphanage
-somewhere until just before. I remember strongly
-that I had to be an idol.#MTLed
+But it's not like I don't remember anything at
+all. Right up until then, I was in an orphanage
+somewhere, and I remember having this strong wish
+to become an idol.#Ron
```

Comment:
A bit of rewording/tone:
_**It's not like I've forgotten everything... Before
this all happened, I used to live in an orphanage.
I always dreamed of becoming an idol...**_